phone_number.rb

```diff
@@ -0,0 +1,63 @@
+require_relative "./phone_actions"
+
+class PhoneNumber
+  USERNAME = ENV['ACTIONS_USERNAME']
+  PASSWORD = ENV['ACTIONS_PASSWORD']
+
+  def initialize(number)
+    @number = number
+    @number = format_number
+  end
+
+  def text(message)
+    PhoneActions.new(USERNAME, PASSWORD).text!(@number, message)
+  end
+
+  def call(message)
+    PhoneActions.new(USERNAME, PASSWORD).call!(@number, message)
+  end
+
+  def text_and_call(message)
+    PhoneActions.new(USERNAME, PASSWORD).text!(@number, message)
+    PhoneActions.new(USERNAME, PASSWORD).call!(message, message)
+  end
+
+  def number
+    @number
+  end
+
+  def to_i
+    n = @number.gsub!(/[^0-9]/, "")
+    n.to_i
+  end
+
+  private
+
+  def format_number
+    "(#{find_area_code}) #{find_exchange}-#{find_line_number}"
```

Comment:
```suggestion
    @format_number = "(#{find_area_code}) #{find_exchange}-#{find_line_number}"
```
Then you can use @format_number instead of @number in your #text and #call methods instead.